src/SDKs/DBforPostgreSQL/DBforPostgreSQL.Tests/DBforPostgreSQL.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@rohit-joy also the csproj should be named as Post.greSql.Tests.csproj

Reply:
Done